docs/user/security/authentication/index.asciidoc

```diff
@@ -383,21 +383,11 @@ xpack.security.authc.providers:
 
 One of the most popular use cases for anonymous access is when you embed {kib} into other applications and don't want to force your users to log in to view it. If you configured {kib} to use anonymous access as the sole authentication mechanism, you don't need to do anything special while embedding {kib}.
 
-If you have multiple authentication providers enabled, and you want to automatically log in anonymous users when embedding, then you will need to add the `auth_provider_hint=<anonymous-provider-name>` query string parameter to the {kib} URL that you're embedding.
```

Comment:
I agree that we don't need this here anymore, but I still think there's value in documenting this somewhere. The new "Public URL" option is fantastic for most users, but I expect we'll get asked about embedding arbitrary links, not just those that we expose a UI for. What do you think?

Reply:
> I agree that we don't need this here anymore, but I still think there's value in documenting this somewhere. The new "Public URL" option is fantastic for most users, but I expect we'll get asked about embedding arbitrary links, not just those that we expose a UI for. What do you think?

I actually tried to find any user facing doc we have that mentions embedding of anything other than dashboards or visualizations and couldn't find any. That made me think that maybe it's not a frequent use case and hence we have a chance to keep it as an implementation detail.

But I don't have a strong opinion, do you have any particular part of ours docs in mind where this would fit?

Reply:
Hmm, I don't have an existing place in mind, but I could see us adding a new page that explains how to embed Kibana. That would be a good place to describe the correct way to configure `xpack.security.sameSiteCookies` too, since that's a common point of confusion.

I realize what I'm asking blows up the scope of your PR a bit, so we can do that in a followup if you would prefer

Reply:
> I realize what I'm asking blows up the scope of your PR a bit, so we can do that in a followup if you would prefer

I think we can expand the scope of this tiny PR, let me think where this can fit.